README.md

```diff
@@ -7,9 +7,23 @@ Tool for:
 
 ## Usage
 
+The command line tool provides two sub commands: `index` and `view` for the operations
+mentioned above. The basic usage format is:
+```
+$ java -jar atlantool.jar index <bam-path>
+
+$ java -jar atlantool.jar view <bam-path> -n <qname-to-search>
+or
+$ java -jar atlantool.jar view <bam-path> -f <file-containing-qnames>
```

Comment:
Let's make the examples consistent - we should either use JAR everywhere or binary everywhere. Right now this example uses JAR, while "Detailed example" uses binary.

Reply:
Done